src/main/groovy/com/palantir/gradle/javadist/JavaDistributionPlugin.groovy

```diff
@@ -31,18 +32,23 @@ class JavaDistributionPlugin implements Plugin<Project> {
         // force application of java
         project.plugins.apply('java')
 
+        project.configurations.create("goJavaLauncherBinaries")
+        project.dependencies {
+            goJavaLauncherBinaries "com.palantir.launching:go-java-launcher:1.0.0"
```

Comment:
(using a gradle configuration also gives you caching for free which is quite handy)